napari/types.py

```diff
@@ -20,7 +20,7 @@
 from typing_extensions import TypedDict, get_args
 
 if TYPE_CHECKING:
-    import dask.array
+    import dask.array  # noqa ICN001
```

Comment:
Is this `noqa` needed or can we comply?

Reply:
I add a rule to always import `dask.array` as `da`. I could remove this rule. 

In general, I keep this name as magicgui requires that if the type is provided as a string, then a full path needs to be used.

Another option is to move this import outside `TYPE_CHECKING` block to use the shortcut.

Reply:
> In general, I keep this name as magicgui requires that if the type is provided as a string, then a full path needs to be used.

I see; then maybe we can just add a comment to explain that.

Reply:
Comment added. Could you check it?